### ISSUE TITLE
[BUG] PHP Fatal error: Arginfo / zpp mismatch during call of Redis::connect()

### ISSUE THREAD

**Issue by cjzyy** [phpredis/phpredis#2438]
php 8.1.22, PhpRedis 5.3.7 

![image](https://github.com/phpredis/phpredis/assets/23442467/15d4c9ae-84cf-459a-a0d9-132c71db853f)

code 
```php
$redis = new Redis();
//$redis->connect('192.168.2.149', 6379); // ok
$redis->connect('192.168.2.149', 6379, 1, NULL, 0, 0); // show error
$redis->auth(123456);
echo $redis->ping()."\r\n";
```

**Comment by michael-grunder** [comment]
I think this has been fixed in the latest version of PhpRedis (6.0.2).

**Comment by cjzyy** [comment]
> I think this has been fixed in the latest version of PhpRedis (6.0.2).

ok， I will upgrade to check

**Comment by cjzyy** [comment]
upgrade redis extend solved
